src/views/v2/list/event/venue.php

```diff
@@ -31,8 +31,13 @@
 	</span>
 	<span class="tribe-events-calendar-list__event-venue-address">
 		<?php echo esc_html( $address ); ?>
+
+		<?php if ( $venue->city ) : ?>
+			<?php echo esc_html( $venue->city . $separator ) ?>
+		<?php endif; ?>
+		
 		<?php if ( $append_after_address ) : ?>
```

Comment:
a lot of open and close php tags, let's make this under one open and close tag

Reply:
I've refactored this to be under one open and close tag